spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/AbstractRabbitListenerContainerFactory.java

```diff
@@ -295,6 +301,26 @@ public void setBeforeSendReplyPostProcessors(MessagePostProcessor... beforeSendR
 		this.beforeSendReplyPostProcessors = beforeSendReplyPostProcessors;
 	}
 
+	/**
+	 * Set a {@link RetryTemplate} to use when sending replies.
+	 * @param retryTemplate the template.
+	 * @since 2.0.6
+	 * @see #setReplyRecoveryCallback(RecoveryCallback)
```

Comment:
How about to also add a `@see AbstractAdaptableMessageListener#setReplyRecoveryCallback(RecoveryCallback)` for better navigation?